modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java

```diff
@@ -1035,7 +1035,31 @@ public void testAddObjectEntryWithFormulaObjectField() throws Exception {
 					).build())
 			).build());
 
-		ObjectEntry objectEntry = _addObjectEntry(
+		ObjectField objectField2 = _addCustomObjectField(
+			new FormulaObjectFieldBuilder(
+			).labelMap(
+				LocalizedMapUtil.getLocalizedMap("Able")
+			).name(
```

Comment:
I'm not sure about the field name, I believe that something like `id sum` is better.